### DISCUSSION TITLE
How to find Ancient Temple?

### DISCUSSION THREAD

**Discussion by sleepysalamancer** [KyaniteMods/DeeperAndDarker#124] [Question] [answered]
any tips for how to find an ancient temple? I've explored a few otherside deeplands biomes just flying around on my elytra for about an hour now and no sign of one. Are they pretty rare? Any certain elevation I should be combing?

**Comment by nitrodynamite18** [comment]
They're excessively rare. We will be making them more common soon.

**Comment by NewJumper** [comment, marked as answer]
Ancient temples spawn at y18 and in the deeplands. They are much more common since update 1.2.0
